osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/MBusGatewayService.java

```diff
@@ -374,6 +375,38 @@ private void checkAndHandleIfMbusDeviceNotFound(
     }
   }
 
+  private void checkAndHandleIfMbusDeviceIsInUse(final SmartMeter mbusDevice)
+      throws FunctionalException {
+    if (DeviceLifecycleStatus.IN_USE == mbusDevice.getDeviceLifecycleStatus()) {
+      throw new FunctionalException(
+          FunctionalExceptionType.MBUS_DEVICE_NOT_MOVED_TO_ANOTHER_EMETER,
+          ComponentType.DOMAIN_SMART_METERING,
+          new OsgpException(
+              ComponentType.DOMAIN_SMART_METERING,
+              String.format(
+                  "Mbus device {} not moved to another E meter",
+                  mbusDevice.getDeviceIdentification())));
+    }
+  }
+
+  private void checkAndHandleIfMbusDeviceIsInUse(
+      final SmartMeter mbusDevice, final CoupleMbusDeviceByChannelResponseDto responseDto)
+      throws FunctionalException {
+    if (DeviceLifecycleStatus.IN_USE == mbusDevice.getDeviceLifecycleStatus()) {
+      throw new FunctionalException(
+          FunctionalExceptionType.MBUS_DEVICE_NOT_MOVED_TO_ANOTHER_EMETER,
+          ComponentType.DOMAIN_SMART_METERING,
+          new OsgpException(
+              ComponentType.DOMAIN_SMART_METERING,
+              String.format(
+                  "Mbus device {} found on channel: %d with mbusIdentificationNumber: %s and mbusManufacturerIdentification: %s not moved to another E meter",
```

Comment:
Ben wel benieuwd of zowel de {} als de %s hier werken? Volgens mij werkt de {} niet in de format

Reply:
Nee, dat denk ik ook niet nee